include/dgl/runtime/ndarray.h

```diff
@@ -6,11 +6,15 @@
 #ifndef DGL_RUNTIME_NDARRAY_H_
 #define DGL_RUNTIME_NDARRAY_H_
 
-#include <string>
+#include <dgl/zerocopy_serializer.h>
```

Comment:
`#include "../zerocopy_serializer.h"`. Maybe just put the code in `"serializer.h"`?